core/src/main/java/org/frankframework/http/rest/ApiListener.java

```diff
@@ -137,11 +139,17 @@ public void configure() throws ConfigurationException {
 	private void validateClaimAttribute(String claimAttributeToValidate, String claimAttributeName) throws ConfigurationException {
 		if(StringUtils.isNotEmpty(claimAttributeToValidate)){
 			List<String> invalidClaims = StringUtil.splitToStream(claimAttributeToValidate)
-					.filter(claim -> StringUtil.split(claim, "=").size() != 2)
+					.filter(claim ->
+							StringUtil.splitToStream(claim, "=")
+									.filter(part -> part.length() > 0)
+									.collect(Collectors.toList())
+									.size() != 2
+					)
```

Comment:
Was there an actual issue with this code, that you changed it?
`StringUtil.split()` should already filter out blank strings so I'm surprised by this change.

Reply:
You are absolutely right, I'll reverse this part of the PR.